views/components/revealable_content.erb

```diff
@@ -1,13 +1,13 @@
 <%# locals: (content:) %>
-<div class="revealable-content">
-  <div class="shadow-content flex items-center gap-2">
+<div class="revealable-content group">
+  <div class="shadow-content items-center gap-2 flex group-[.active]:hidden">
```

Comment:
`group-[.active]:hidden` is some Tailwind magic I presume?

Reply:
Actually, these are new CSS features that let you style based on the parent or siblings. Tailwind has a shorter syntax for this.
https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state